eng/Versions.props

```diff
@@ -1,5 +1,7 @@
 <Project>
   <PropertyGroup>
+    <!-- Current previous .NET SDK major version's stable release of MAUI packages -->
+    <MicrosoftMauiPreviousDotNetReleasedVersion>6.0.400</MicrosoftMauiPreviousDotNetReleasedVersion>
```

Comment:
This version will have to be manually updated to the latest released version of .NET MAUI each time a .NET 6 patch goes out.